src/storage/store/node_table.cpp

```diff
@@ -293,7 +293,7 @@ void NodeTable::commit(Transaction* transaction, LocalTable* localTable) {
     for (auto localNodeGroupIdx = 0u; localNodeGroupIdx < localNodeTable.getNumNodeGroups();
          localNodeGroupIdx++) {
         const auto localNodeGroup = localNodeTable.getNodeGroup(localNodeGroupIdx);
-        if (localNodeGroup->getNumDeletedRows(transaction) > 0) {
+        if (localNodeGroup->hasDeletions(transaction) > 0) {
```

Comment:
While this is a valid way of checking if hasDeletions returned `true`, I'm guessing you probably forgot to remove the `> 0`?

Reply:
Yeah. good catch. removed that now.